avm/res/event-hub/namespace/eventhub/main.bicep

```diff
@@ -309,11 +309,11 @@ type captureDescriptionType = {
 }
 
 @export()
-@description('An AVM-aligned type for a managed identity configuration. To be used if both a system-assigned & user-assigned identities are supported by the resource provider.')
+@description('An AVM-aligned type for a managed identity configuration. Supports a muturally exclusive system- & user-assigned identity.')
```

Comment:
What do we mean with "An AVM-aligned type"? Is that intended also for local types?

Reply:
It's AVM-aligned because it has the `managedIdentities` schema & logic (e.g., with the `system-assigned` boolean. That's why I kept the wording. The same is true for other modules with custom types